CMakeLists.txt

```diff
@@ -55,3 +55,8 @@ endif()
 if(BUILD_TESTING AND QTRANSPORT_BUILD_TESTS)
   add_subdirectory(cmd)
 endif()
+
+install(TARGETS ${PROJECT_NAME}
+        LIBRARY DESTINATION lib)
```

Comment:
Could you give a `FRAMEWORK DESTINATION` too? Annoyingly it is not optional for CMake and causes an error when building to Apple frameworks.